src/main/java/com/modrinth/minotaur/TaskModrinthUpload.java

```diff
@@ -98,19 +100,7 @@ public void apply() {
 
 			// Attempt to automatically resolve the loader if none were specified.
 			if (ext.getLoaders().get().isEmpty() && ext.getDetectLoaders().get()) {
-				Map<String, String> pluginLoaderMap = new HashMap<>();
-				pluginLoaderMap.put("net.minecraftforge.gradle", "forge");
-				pluginLoaderMap.put("fabric-loom", "fabric");
-				pluginLoaderMap.put("org.quiltmc.loom", "quilt");
-				pluginLoaderMap.put("org.spongepowered.gradle.plugin", "sponge");
-				pluginLoaderMap.put("io.papermc.paperweight.userdev", "paper");
-
-				pluginLoaderMap.forEach((plugin, loader) -> {
-					if (pluginManager.hasPlugin(plugin)) {
-						getLogger().debug("Adding loader {} because plugin {} was found.", loader, plugin);
-						ext.getLoaders().add(loader);
-					}
-				});
+				autoDetectLoaders(ext, pluginManager);
```

Comment:
Any reason in particular to separate these out?

Reply:
Nothing beyond that I was making an already massive method bigger. Which I guess it's just part of my code style that I would start breaking it up. I can inline it if that's wanted.

Reply:
If you think this is a large method, you should see labrinth :D https://github.com/modrinth/labrinth/blob/master/src/routes/projects.rs#L388-L1140

I would rather it be inlined since there's only one usage.